app/js/lib/foxbox/recipes.js

```diff
@@ -54,9 +54,8 @@ export default class Recipes {
       'POST',
       { getters: [{ kind: 'ThinkerbellRuleSource' }] }
     )
-    .then((services) => {
       // Mark getters and setters with more friendly names.
-      return services.map((service) => {
+      .then((services) => services.map((service) => {
```

Comment:
__question:__ When closing bracket is moved to a separate line above it looks nicer when ```.then``` is aligned with it, what do you think?